osu.Game/Beatmaps/ControlPoints/DifficultyControlPoint.cs

```diff
@@ -40,13 +47,25 @@ public double SliderVelocity
             set => SliderVelocityBindable.Value = value;
         }
 
+        /// <summary>
+        /// Whether or not slider ticks should be generated at this control point.
+        /// This exists for backwards compatibility with maps that abuse NaN slider velocity behavior on osu!stable (e.g. /b/2628991).
+        /// </summary>
+        public bool GenerateTicks
+        {
+            get => GenerateTicksBindable.Value;
+            set => GenerateTicksBindable.Value = value;
+        }
```

Comment:
Not too happy about this existing in `DifficultyControlPoint`..

Reply:
I couldn't really think of anything better since the slider tick disabling is a direct effect of the slider velocity itself.